release_tools/changelog.py

```diff
@@ -211,13 +212,33 @@ def create_changelog_entry_content(title, category, author=None, issue=None,
     return content
 
 
-def write_changelog_entry(dirpath, title, content, overwrite=False):
-    """Store the contents of an entry in a file."""
-
+def validate_changelog_content(content, run_editor=True):
     if not content:
         msg = "Aborting due to empty entry content"
         raise click.ClickException(msg)
 
+    while True:
+        try:
+            data = yaml.safe_load(content)
+            return content
+        except yaml.YAMLError as exc:
+            pm = exc.problem_mark
+            click.echo("Error: File has an issue on line {} at position {}".format(pm.line, pm.column))
```

Comment:
As there's not file created yet, maybe it's better to say `Invalid format on ...`